helm/templates/configmap.yaml

```diff
@@ -0,0 +1,65 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ template "authelia.fullname" . }}-config
+  labels:
+{{ include "authelia.labels" . | indent 4 }}
+data:
+  config.yml: |-
+    port: 9091
+
+    logs_level: {{ .Values.authelia.logLevel }}
+
+    default_redirection_url: {{ .Values.authelia.redirectionURL }}
+
+    totp:
+      issuer: {{ .Values.authelia.totpIssuer }}
+
+    {{- if .Values.authelia.duo.enabled }}
+    duo_api:
+{{ toYaml .Values.authelia.duo.config | indent 6 }}
+    {{- end }}
+
+    authentication_backend:
+      {{- if .Values.authelia.ldap.enabled }}
+      ldap:
+{{ toYaml .Values.authelia.ldap.config | indent 8 }}
+      {{- else }}
+      file:
+        path: /var/lib/authelia/users_database.yml
+      {{- end }}
+
+    access_control:
+      default_policy: {{ .Values.authelia.accessControl.defaultPolicy }}
+      rules:
```

Comment:
For the future, I had in mind to maybe provide an extension to kube ingress configuration to define the level of authentication required by the ingress directly in the ingress config instead of Authelia config. Just an idea as I ready the chart. What do you think?

Reply:
I'm not sure I follow, but are you talking about consuming annotations in ingresses? If so, then yeah, that's a nice idea, but on the general people are starting to move away from annotating and into CRDs where this kind of stuff should belong. Case in point : traefik 2.0 or istio

Reply:
Good idea.